### PR TITLE
Fixed SSDP scan function and added interfaces function to ssdp.py

### PR DESCRIPTION
**Pull request by gitaaronc**

### CODE REVIEW

File: pywemo/ssdp.py [hunk 1]
```diff
@@ -146,13 +146,13 @@ def description(self):
                     UPNPEntry.DESCRIPTION_CACHE[url] = None
 
             except requests.RequestException:
-                logging.getLogger(__name__).error(
+                logging.getLogger(__name__).exception(
```
Comment (kk7ds) [inline pywemo/ssdp.py:149]: Why are you changing these?

File: pywemo/ssdp.py [hunk 2]
```diff
@@ -133,7 +133,7 @@ def description(self):
 
         if url not in UPNPEntry.DESCRIPTION_CACHE:
             try:
-                xml = requests.get(url, timeout=10).text
```
Comment (kk7ds) [inline pywemo/ssdp.py:136]: Why remove this timeout? This could block for a long time otherwise.

File: pywemo/ssdp.py [hunk 3]
```diff
@@ -215,40 +219,47 @@ def scan(st=None, timeout=DISCOVER_TIMEOUT, max_entries=None, match_mac=None):
     start = calc_now()
 
     try:
+        """
         sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-
+        sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        sock.setsockopt(socket.IPPROTO_IP, socket.IP_MULTICAST_TTL, 2)
+        sock.bind(('192.168.4.134', 8012))
```
Comment (kk7ds) [inline pywemo/ssdp.py]: This won't work for anyone but you :)

Comment (gitaaronc) [inline pywemo/ssdp.py]: Those three lines are in a comment block.

File: pywemo/ssdp.py (hunk 2 above)
Comment (gitaaronc) [inline pywemo/ssdp.py:136]: Also merged from the master

File: pywemo/ssdp.py (hunk 1 above)
Comment (gitaaronc) [inline pywemo/ssdp.py:149]: I actually didn't change those lines. Those were merged from the master.

Comment (kk7ds) [inline pywemo/ssdp.py:149]: Then you need to rebase or something?

File: pywemo/ssdp.py (hunk 3 above)
Comment (kk7ds) [inline pywemo/ssdp.py]: Is that helpful in some way? It's just a bit of commented out code that won't run on anyone's system. I would just remove it. Commented-out code is evil :)

Comment (gitaaronc) [inline pywemo/ssdp.py]: Will do.

File: pywemo/ssdp.py (hunk 1 above)
Comment (gitaaronc) [inline pywemo/ssdp.py:149]: Perhaps. I don't use git much. I only forked pywemo last night.

**Comment by gitaaronc** [comment]
Can you close/cancel this pull request? I have create a new one.

**Pull request closed without merging**